App/Sagas/TextileSagas.ts

```diff
@@ -93,26 +94,43 @@ export function * updateNodeOverview ( action: ActionType<typeof TextileNodeActi
   }
 }
 
+export function * onboardedSuccess(action: ActionType<typeof PreferencesActions.onboardedSuccess>) {
```

Comment:
Seems like we can just check for and possibly create the default thread after starting the node, before telling the rest of the app that the node is started. Also, that way we don't have to tell the app to refresh threads etc.

Reply:
okay, added a ticket to move that https://github.com/textileio/textile-mobile/issues/428